File: src/Moryx.ControlSystem.Processes.Endpoints/Models/ProcessHolderPositionModel.cs
// Copyright (c) 2025, Phoenix Contact GmbH & Co. KG
// Licensed under the Apache License, Version 2.0

namespace Moryx.ControlSystem.Processes.Endpoints;

public class ProcessHolderPositionModel
{
    public long Id { get; set; }

    public long GroupId { get; set; }

    public string Name { get; set; }

    public int Position { get; set; }

    public string Activity { get; set; }

    public string Order { get; set; }

    public string Process { get; set; }

    public bool IsEmpty { get; set; }
}

File: src/Moryx.ControlSystem.ProcessEngine.Web/app/src/app/models/process-holder-position-model.ts
/*
 * Copyright (c) 2026 Phoenix Contact GmbH & Co. KG
 * Licensed under the Apache License, Version 2.0
*/

export interface ProcessHolderPosition {
    id: number;
    groupId: number;
    name: string;
    position: number;
    activity: string;
    order: string;
    process: string;
    isEmpty: boolean;
}
